docs/attributes-registry/db.md

```diff
@@ -166,6 +166,16 @@
 | `ExecuteJavaScript` | execute_javascript |
 <!-- endsemconv -->
 
+<!-- semconv registry.db(omit_requirement_level,tag=tech-specific-cosmosdb) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| [`rntbd.url`](../attributes-registry/db.md) | string | CosmosDB server url contacted | `rntbd://some-url/` |
+| [`rntbd.status_code`](../attributes-registry/db.md) | string | Status Code returned by cosmos DB server | `200` `404` etc. |
+| [`rntbd.sub_status_code`](../attributes-registry/db.md) | string | Sub Status Code which tells more about the failure | `20001` `20002` etc |
+| [`rntbd.*`](../attributes-registry/db.md) | string | Cosmos DB specific information, it can vary with different flavours of SDK |  |
+| [`rntbd.pipeline.*`](../attributes-registry/db.md) | double | Cosmos specific attribute with detailed information about the request pipeline | |
```

Comment:
what does the `*` mean?

Reply:
`* ` means it can have any cosmos db specific attribute. It might vary with SDK flavour, there is not fix list of attributes we would want to have.

Reply:
what's the point in documenting them then? how do you want people to use it and what stability for `*` you can guarantee?